lib/chitter.rb

```diff
@@ -0,0 +1,34 @@
+require './lib/db_connection'
+require './lib/peep'
+require './lib/user'
+require './lib/tag'
+
+class Chitter
```

Comment:
It's great that you've group the all methods on the chitter class, identifying the users, peeps and tags now belong to chitter